android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -95,26 +100,48 @@ private static void doLaunch(Context context, HashMap<String, Object> extraData)
 
 
     public static void launch(Context context, HashMap<String, Object> extraData) {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-            PermissionListener listener = new PermissionListener() {
-                @Override
-                public boolean onRequestPermissionsResult(int i, String[] strings, int[] results) {
-                    if (results.length > 0 && results[0] == PackageManager.PERMISSION_GRANTED) {
-                        doLaunch(context, extraData);
-                    }
-
-                    return true;
+        List<String> permissionsList = new ArrayList<String>();
+
+
+        PermissionListener listener = new PermissionListener() {
+            @Override
+            public boolean onRequestPermissionsResult(int i, String[] strings, int[] results) {
+                if (results.length > 0 && results[0] == PackageManager.PERMISSION_GRANTED) {
+                    JitsiMeetLogger.w(TAG + " POST_NOTIFICATIONS permission granted");
                 }
-            };
+
+                return true;
+            }
+        };
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            permissionsList.add(POST_NOTIFICATIONS);
 
             JitsiMeetActivityDelegate.requestPermissions(
```

Comment:
Don't immediately check the permissions here, do it only if the array is not empty.